NHLGames/Utilities/ImageFetcher.vb

```diff
@@ -33,11 +33,17 @@ Namespace Utilities
             Dim files = New String() {"nhlgames", "alt", "att", "cbc", "espn", "fs", "msg", "nbc", "nesn", "rds", "root", "sn", "tsn", "tvas", "wgn", "broken", "cam", "cam_left", "cam_right", "cam_ref", "cam_star", "cam3way", "cam6way", "date_d", "date_l", "down_d", "down_l", "download", "left_d", "left_l", "live_0", "live_1", "live_2", "mpc", "mpv", "refresh_d", "refresh_l", "right_d", "right_l", "up_d", "up_l", "vlc", "ana_d", "ana_l", "ari_d", "ari_l", "bos_d", "bos_l", "buf_d", "buf_l", "car_d", "car_l", "cbj_l", "cbj_d", "cgy_d", "cgy_l", "chi_d", "chi_l", "col_d", "col_l", "dal_d", "dal_l", "det_d", "det_l", "edm_d", "edm_l", "fla_d", "fla_l", "lak_d", "lak_l", "min_d", "min_l", "mtl_d", "mtl_l", "nhl_d", "nhl_l", "njd_l", "njd_d", "nsh_l", "nsh_d", "nyi_d", "nyi_l", "nyr_d", "nyr_l", "ott_d", "ott_l", "phi_d", "phi_l", "pit_d", "pit_l", "sjs_d", "sjs_l", "stl_d", "stl_l", "tbl_d", "tbl_l", "tor_d", "tor_l", "van_d", "van_l", "vgk_d", "vgk_l", "wpg_d", "wpg_l", "wsh_d", "wsh_l"}
 
             For Each f In files
-                Using s As Stream = myAssembly.GetManifestResourceStream($"NHLGames.{f.ToLower()}.svg")
-                    If s Is Nothing Then Return
-                    s.Position = 0
-                    Dim x = (SvgDocument.Open(Of SvgDocument)(s)).Draw()
-                    x.Save($"c:/output/{f}.png")
+                Using s As Stream = myAssembly.GetManifestResourceStream($"NHLGames.{f.ToLower()}.png")
```

Comment:
did you find a bug here
if i remember correctly, and if the images above are still svg(s) they were set that way so they get revaluated at every launch based on user interface custom ratio, png were not so great on different display